### PR TITLE
feat: Update Release notes for 10.1 Jacinto Foundational SDK

### PR DESCRIPTION
**Pull request by aniket-l**
Update release docs for AM67, AM68, AM69
Release_notes: tag/info for meta-tisdk layer
Yocto_layer_Configuration: yocto config file for the build

### CODE REVIEW

**Review by uditkumarti — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]
LGTM!

File: source/devices/J7_Family/linux/Release_Specific_Release_Notes.rst [hunk 1]
```diff
@@ -105,10 +105,6 @@ U-Boot
 | Compiler Information: arm-oe-eabi-gcc (GCC) 13.3.0, aarch64-oe-linux-gcc (GCC) 13.3.0
 |
 
-.. note::
-
```
Comment (praneethbajjuri) [inline source/devices/J7_Family/linux/Release_Specific_Release_Notes.rst:109]: Why is this note removed? I understand that currently there might not be any appended patches in meta-tisdk later, but in future if they do exists a note like this is still valid. It's a disclaimer note saying "may contain", looks harmless to me to keept it .

Comment (cshilwant) [reply source/devices/J7_Family/linux/Release_Specific_Release_Notes.rst:109]: @praneethbajjuri We have come up with a policy to not have any patches in meta-tisdk layer
Right @aniket-l :wink:

Comment (cshilwant) [reply source/devices/J7_Family/linux/Release_Specific_Release_Notes.rst:109]: But yeah, we can add the above disclaimer for meta-edgeai 

- https://git.ti.com/cgit/edgeai/meta-edgeai/tree/recipes-bsp/u-boot/u-boot-ti-staging?h=scarthgap
- https://git.ti.com/cgit/edgeai/meta-edgeai/tree/recipes-kernel/linux/linux-ti-staging?h=scarthgap

Comment (StaticRocket) [reply source/devices/J7_Family/linux/Release_Specific_Release_Notes.rst:109]: Yeah, if PSDK is making an effort to no longer patch these components then there's no need to note this explicitly here. I do think it's worth noting in a Yocto specific section that one can check what patches are being applied to the kernel using bitbake to dump the `SRC_URI` variable for a given release.

**Review by StaticRocket — APPROVED** [review]
This is fine to me right now. Leaving merge to @praneethbajjuri in case we want to push that comment a little more.

**Pull request merged**